Add --disable-wbar-config to support "gtk" USE flag.
Setting it will disable building and installing the gtk-based
config program.

Patch by Kevin McCarthy <signals@gentoo.org>

--- configure.ac
+++ configure.ac
@@ -60,9 +60,15 @@
 AC_SUBST(modules_CFLAGS)
 AC_SUBST(modules_LIBS)
 
-PKG_CHECK_MODULES(gui_modules, [gtk+-2.0 libglade-2.0])
-AC_SUBST(gui_modules_CFLAGS)
-AC_SUBST(gui_modules_LIBS)
+AC_ARG_ENABLE(wbar-config, AS_HELP_STRING([--disable-wbar-config],
+	[build without the GTK+ configuration interface]),[use_wbar_config=$enableval],
+	[use_wbar_config=yes])
+if test "$use_wbar_config" = "yes"; then
+	PKG_CHECK_MODULES(gui_modules, [gtk+-2.0 libglade-2.0])
+	AC_SUBST(gui_modules_CFLAGS)
+	AC_SUBST(gui_modules_LIBS)
+fi
+AM_CONDITIONAL([WBAR],[test "$use_wbar_config" = "yes"])
 
 CPPFLAGS="${CPPFLAGS} -Wall -Wunused -Werror -g -O2"
 
--- src/Makefile.am
+++ src/Makefile.am
@@ -1,6 +1,10 @@
 CLEANFILES = *~
 
+if WBAR
 bin_PROGRAMS = wbar wbar-config
+else
+bin_PROGRAMS = wbar
+endif
 
 wbar_SOURCES = \
 $(top_srcdir)/src/core/Bar.cc \
--- Makefile.am
+++ Makefile.am
@@ -1,4 +1,7 @@
 CLEANFILES = *~ po/*~
+if WBAR
 SUBDIRS = src po pixmaps etc share
-
+else
+SUBDIRS = src po pixmaps etc
+endif
 
